testutils/testutils.go

```diff
@@ -0,0 +1,60 @@
+package testutils
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// Output directory for testing and benchmark artifacts
+func GetTestOutDir() string {
+	out, _ := exec.Command("git", "rev-parse", "--show-toplevel").CombinedOutput()
+	repoRoot := string(out)
+	prefix := filepath.Join(strings.TrimSpace(repoRoot), "output")
+	return prefix
+}
+
+var (
+	ArchivesDir     = filepath.Join("test-artifacts", "archives")
+	BundlesRoot     = filepath.Join("test-artifacts", "oci-bundles")
+	OutputDirFormat = filepath.Join("test-artifacts", "runs", "%s")
+	RefOciSpecsPath = filepath.Join(BundlesRoot, "config.json")
+	StateDir        = "/run/containerd"
```

Comment:
Probably want to use a different path that the default one. maybe `/run/containerd-benchmark-test`?